drivers/png/SCsub

```diff
@@ -37,7 +37,7 @@ if env["builtin_libpng"]:
     env_thirdparty.disable_warnings()
     env_thirdparty.add_source_files(thirdparty_obj, thirdparty_sources)
 
-    if env["arch"].startswith("arm"):
+    if env["arch"].startswith("arm") or env["arch"].startswith("aarch"):
```

Comment:
`aarch64` is not a valid value for arch, it should be either `arm32` or `arm64` (so both will already trigger this condition), see - https://github.com/godotengine/godot/blob/64150060f89677eaf11229813ae6c5cf8a873802/platform_methods.py#L82C1-L82C1 for valid arch values.